legoshop.py

```diff
@@ -416,12 +451,16 @@ def __process_select_lego_set(self, lego_set):
             lego_set=lego_set))
         setno_field = self.wait.until(
             EC.element_to_be_clickable(
-                (By.CSS_SELECTOR, '.product-search input[ng-model=productNumber]'))
+                (By.XPATH, "//input[@data-cy = 'enter-set-number-search']"))
```

Comment:
It looks like you prefer XPath over CSS selectors. I feel this is less readable, harder to understand, harder to maintain. Would you be willing to change all calls to CSS selectors?